src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -6,19 +6,21 @@
 public class CliSyntax {
 
     /* Prefix definitions */
-    public static final Prefix PREFIX_NAME = new Prefix("n/");
-    public static final Prefix PREFIX_PHONE = new Prefix("p/");
-    public static final Prefix PREFIX_EMAIL = new Prefix("e/");
-    public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
-    public static final Prefix PREFIX_TAG = new Prefix("t/");
-    public static final Prefix PREFIX_MATRIC_NUMBER = new Prefix("m/");
+    public static final Prefix PREFIX_NAME = new Prefix("n|");
+    public static final Prefix PREFIX_PHONE = new Prefix("p|");
+    public static final Prefix PREFIX_EMAIL = new Prefix("e|");
+    public static final Prefix PREFIX_ADDRESS = new Prefix("a|");
+    public static final Prefix PREFIX_TAG = new Prefix("t|");
+    public static final Prefix PREFIX_MATRIC_NUMBER = new Prefix("m|");
```

Comment:
good job changing the convention to `|` as it will help with possible bugs that could arise with `/`